spring-cloud-alibaba-sentinel/src/main/java/com/alibaba/cloud/sentinel/SentinelWebAutoConfiguration.java

```diff
@@ -61,44 +57,48 @@
 	private Optional<UrlCleaner> urlCleanerOptional;
 
 	@Autowired
-	private Optional<UrlBlockHandler> urlBlockHandlerOptional;
+	private Optional<BlockExceptionHandler> blockExceptionHandlerOptional;
 
 	@Autowired
 	private Optional<RequestOriginParser> requestOriginParserOptional;
 
-	@PostConstruct
-	public void init() {
-		urlBlockHandlerOptional.ifPresent(WebCallbackManager::setUrlBlockHandler);
-		urlCleanerOptional.ifPresent(WebCallbackManager::setUrlCleaner);
-		requestOriginParserOptional.ifPresent(WebCallbackManager::setRequestOriginParser);
+	@Autowired
+	private Optional<SentinelWebInterceptor> sentinelWebInterceptorOptional;
+
+	@Override
+	public void addInterceptors(InterceptorRegistry registry) {
+		if (!sentinelWebInterceptorOptional.isPresent()) {
+			return;
+		}
+		SentinelProperties.Filter filterConfig = properties.getFilter();
+		registry.addInterceptor(sentinelWebInterceptorOptional.get())
```

Comment:
cause NPE when `sentinelWebInterceptorOptional.get()` is null

Reply:
The previous code `sentinelWebInterceptorOptional.isPresent` can be avoided NPE